app/build.gradle.kts

```diff
@@ -176,9 +177,33 @@ dependencies {
     androidTestUtil(libs.androidx.orchestrator)
 }
 
-tasks.named("preBuild") { dependsOn("copyLicenseToAssets") }
-tasks.register<Copy>("copyLicenseToAssets") {
+tasks.named("preBuild") {
+    dependsOn(copyLicenseToAssets)
+    dependsOn(fetchBuiltinBridges)
```

Comment:
Haven't had a chance to run this yet, but doesn't this happen every single time someone builds Orbot? Seems like doing something to limit the file is important. Copying the LICENSE file to the disk everytime a build runs doesn't bother me in the same way because it's a tiny file that's already on the disk.